package.json

```diff
@@ -87,9 +84,10 @@
     "sass": "^1.58.3",
     "sass-loader": "^13.3.2",
     "sass-resources-loader": "^2.2.5",
```

Comment:
You should also be able to remove `sass-resources-loader`

Reply:
Done

Reply:
@Judahmeek 
We need to keep `sass-resources-loader` to be used in this project. We want to demonstrate it. By changing to Tailwind, its old usage is no longer relevant, but I should be able to load another sass file.

So, this PR should remain open till I get it to work.